src/app/view-event/page.tsx

```diff
@@ -1,19 +1,100 @@
 'use client'
 import { UUID } from 'crypto'
-import EventView from '@/components/EventView'
+
 import { Suspense, useEffect, useState } from 'react'
-import EventCard from '@/components/EventCard'
 import { useSearchParams } from 'next/navigation'
 
+import EventView from '@/components/EventView'
+import EventCard from '@/components/EventCard'
+import Grid from '@/components/AvailabilityGrid'
+import Responses from '@/components/Responses'
+
+// Placeholder data for responders, including names and availability times
+interface Attendee {
```

Comment:
you should be able to find this in attendeesUtil.ts that Zhenyu added. Should be the same. If so, please use that.

Reply:
should be updated in the next commit